packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -182,4 +204,36 @@ export class BaseButton extends BaseComponent<IButtonProps, {}> implements IButt
         null
       );
   }
+
+  protected _onRenderMenuIcon = (props: IButtonProps): JSX.Element => {
```

Comment:
Should have been clearer, yes, no change in functionality, but wanted to propose to not use lambda functions but autobind for consistency? :)